### ISSUE TITLE
Test suite failure

### ISSUE THREAD

**Issue by snoyberg** [mvoidex/hdocs#3]
Looks like a file is missing from the sdist tarball

```
$ ./dist/build/test/test

tests/Test.hs:11:8:
    Could not find module ‘HDocs.Module’
    Use -v to see a list of the files searched for.

tests/Test.hs:12:8:
    Could not find module ‘HDocs.Haddock’
    Use -v to see a list of the files searched for.
```

**Comment by mvoidex** [comment]
`HDocs.Module` and `HDocs.Haddock` are `hdocs` exposed modules
How to reproduce this issue?

**Comment by snoyberg** [comment]
Cam you try cabal unpack and see if it works?

**Comment by mvoidex** [comment]
`cabal unpack; cd hdocs-0.4.1.0; cabal configure; cabal build; cabal test` runs fine

**Comment by snoyberg** [comment]
I think your tests are presuming that the library is *registered* before being installed. If you run `ghc-pkg unregister hdocs` and then run `cabal test`, I think it will fail.

**Comment by mvoidex** [comment]
Yes, it fails. Does this mean, that test can't depend on library?

**Comment by snoyberg** [comment]
I'm not exactly sure what's going on in the test suite to be honest. It seems like the test itself is running GHC.

**Comment by mvoidex** [comment]
Sorry, misunderstood issue, i'll fix it soon

**Comment by mvoidex** [comment]
Must be fixed in `0.4.1.1` with https://github.com/mvoidex/hdocs/commit/b54e20651276483e11853572b97beb8b9a2f5154 and https://github.com/mvoidex/hdocs/commit/d31e79addb50194e36b179a60461d52e984149d8

**Comment by mvoidex** [comment]
Seems to work after `ghc-pkg unregister` on Win and Ubuntu.
I'll close this issue, let me know if smth goes wrong.
